q2_quality_control/decontam.py

```diff
@@ -14,36 +14,49 @@
+def _check_prev_inputs(table, metadata, prev_control_column,
+                       prev_control_indicator, method):
+    if prev_control_column is None or prev_control_indicator is None:
+        raise ValueError('For the ' + str(method) + 'method '
+                         'please check that input parameters'
+                         ' --prev-control-column and '
+                         '--p-prev-control-indicator are being utilized')
     else:
-        if prev_control_indicator not in list(
-                metadata[prev_control_column]):
-            raise ValueError('No control values found, please select '
-                             'from:\n' +
-                             str(', '.join(
-                                 metadata[prev_control_column]
-                                 .unique())))
+        if prev_control_column not in metadata.columns:
+            raise ValueError('Prevalence column not found, please '
+                             'select from:\n'
+                             + str(', '.join(metadata.columns)))
         else:
-            prev_controls = metadata.loc[
-                metadata[prev_control_column] == prev_control_indicator]
-            prev_sample_names = prev_controls.index.values
-            indic = 0
-            for name in prev_sample_names:
-                if name in table.index.values:
-                    indic = indic + 1
-            if indic < 5:
-                '''raise ValueError('At least 5 Control Samples needed '
-                                 + str(indic) + ' found')'''
-                print("We recommend 5 control samples - " +
-                      str(indic) + " found")
+            if prev_control_indicator not in list(
+                    metadata[prev_control_column]):
+                raise ValueError('No control values found, please select '
+                                 'from:\n' +
+                                 str(', '.join(metadata[prev_control_column]
+                                               .unique())))
             else:
-                print("All appropriate inputs are found")
+                prev_controls = metadata.loc[
+                    metadata[prev_control_column] == prev_control_indicator]
+                prev_sample_names = prev_controls.index.values
+                indic = 0
+                for name in prev_sample_names:
+                    if name in table.index.values:
+                        indic = indic + 1
+                if indic < 5:
+                    print("We recommend 5 control samples - " +
+                          str(indic) + " found")
+
+
+def _check_freq_inputs(metadata, freq_concentration_column, method):
+    if freq_concentration_column is None:
+        raise ValueError('For the ' + str(method) + 'method please check '
+                         'that input parameter'
+                         ' --p-freq-concentration-column is '
+                         'being utilized')
+    else:
```

Comment:
Delete for same reason as above

Reply:
addressed in commit [f31210b](https://github.com/qiime2/q2-quality-control/pull/88/commits/f31210b1b4524e53c8545435910ecb106d40fc8d)